ccp/ses-ansible/roles/ses/tasks/deepsea_setup.yml

```diff
@@ -0,0 +1,104 @@
+---
+
+- name: Install deepsea
+  package:
+    name: "deepsea"
+
+- name: Drop deepsea config(s)
+  template:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+  with_items:
+    - src: "master_minion.sls.j2"
+      dest:  "/srv/pillar/ceph/master_minion.sls"
+    - src: "deepsea_minions.sls.j2"
+      dest: "/srv/pillar/ceph/deepsea_minions.sls"
+
+- name: "Ensure DEV_ENV setting in globals"
+  lineinfile:
+    dest: "/srv/pillar/ceph/stack/global.yml"
+    state: present
+    create: yes
+    regexp: "^DEV_ENV"
+    line: "DEV_ENV: {{ ses_dev_env_setting | lower }}"
+    owner: salt
+    group: salt
+    mode: 0644
+
+- name: Run stage 0 - prepare
+  command: "salt-run state.orch ceph.stage.prep"
+  register: stage0
+  changed_when: false
+  ignore_errors: yes
+
+- name: Show stage 0 (salt-run state.orch ceph.stage.prep) results
+  debug:
+    var: stage0.stdout_lines
+    verbosity: 1
+
+- name: Run stage 1 - discovery
+  command: "salt-run state.orch ceph.stage.discovery"
+  register: stage1
+  changed_when: false
+
+- name: show stage 1 (salt-run state.orch ceph.stage.discovery) results
+  debug:
+    var: stage1.stdout_lines
+    verbosity: 1
+
+- name: Drop policy file
+  copy:
+    src: "policy.cfg"
+    dest: "/srv/pillar/ceph/proposals/policy.cfg"
+
+- name: Drop config for single node deployment
+  template:
+    src: "global.conf.j2"
+    dest: "/srv/salt/ceph/configuration/files/ceph.conf.d/global.conf"
+
+- name: Chanage RGW default port
```

Comment:
nit: change.

Reply:
Yes, I remember seeing this in original ses-ansible change but was able to track down at that time. Thanks for noticing it and will fix in next update.